index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="main-container">
+        <div class="nav-bars">
+            <div class="logo-show">
+              <h1>My Logo</h1>
+            </div>
+            <div class="menu">
+              <div><img src="photo/navs/Union.png" alt=""></div>
+            </div>
+        </div>
+   
+        <div class="myself">
+            <div class="greeting">
+               <h1>I'm Benjamin Glad to see you!</h1>
+            </div>
+
+            <div class="bio">
+              <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+
+            <div class="link">
+             <p>LET'S CONNECT</p>
+            </div>
+        
+            <div class="icon">
+               <img src="photo/logos/twitter.jpg" alt="">
+               <img src="photo/navs/Linkedin-icon.jpg" alt="">
```

Comment:
Great job @osbentech 🤝

- [ ] [Optional] To allow for screen readers which are made possible by the use of semantic tags, It would be better if you put this section in a `<section>` tag.